L/libwebsockets/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "libwebsockets"
+version = v"4.3.3"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/warmcat/libwebsockets.git", "5102a5c8d6110b25a01492fcf96fb668b13dd6e7"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libwebsockets
+
+mkdir build && cd build
+
+cmake .. -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DZLIB_LIBRARY=${libdir}/libz.${dlext} \
+    -DZLIB_INCLUDE_DIR=${includedir} \
+    -DLWS_WITH_HTTP2=1 \
+    -DLWS_WITHOUT_TESTAPPS=1
+
+make -j${nproc}
+make install
```

Comment:
https://docs.binarybuilder.org/stable/build_tips/#CMake-builds
```suggestion
cmake -B build \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    -DZLIB_LIBRARY=${libdir}/libz.${dlext} \
    -DZLIB_INCLUDE_DIR=${includedir} \
    -DLWS_WITH_HTTP2=1 \
    -DLWS_WITHOUT_TESTAPPS=1

cmake --build build --parallel ${nproc}
cmake --install build
```